### ISSUE TITLE
Composer update got error

### ISSUE THREAD

**Issue by minhna** [basvandorst/StravaPHP#5]
> Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - Installation request for basvandorst/stravaphp 1.0.0 -> satisfiable by basvandorst/stravaphp[1.0.0].
    - basvandorst/stravaphp 1.0.0 requires league/oauth2-client dev-master -> no matching package found.

Could you fix that?
Thank you.

**Comment by minhna** [comment]
If I change the version to `dev-master`, still got the error:

> Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - Installation request for basvandorst/stravaphp dev-master -> satisfiable by basvandorst/stravaphp[dev-master]
.
    - basvandorst/stravaphp dev-master requires league/oauth2-client master -> no matching package found.

**Comment by minhna** [comment]
I think, in the composer.json, you modify

```
 "require": {
        "educoder/pest": "1.0.0",
        "league/oauth2-client": "master"
    },
```

to
```
 "require": {
        "educoder/pest": "1.0.0",
        "thephpleague/oauth2-client": "master"
    },
```

it will solve the problem.

**Comment by basvandorst** [comment]
@minhna thanks for your suggestion, but the name "league" was right. There was something wrong with the lock-in on the old dev-master. Please checkout the new master version or the 1.0.1 version

**Comment by Bendihossan** [comment]
@basvandorst If that packagist namespace is correct then the README wants updating as it currently says to use ``basvandorst/stravaphp``: https://github.com/basvandorst/StravaPHP/blob/master/README.md#composer-package

PS. Thanks for the library!

**Comment by Bendihossan** [comment]
Ignore my previous comment, I just got completely confused sorry.

**Comment by minhna** [comment]
changed the version to 1.0.1 and it works.
Now I'm having an issue with `lib-curl`.

**Comment by basvandorst** [comment]
It think that's a development environment issue. Make sure you've installed the curl extension.
Please search for curl install windows/linux howto's :)

**Comment by roberto-butti** [comment]
I had the same issue.
In my case, to fix I made 2 steps: I added some packages in composer.json, I added curl extension in php.
In composer.json I added these lines in "require" sections:

    "basvandorst/StravaPHP": "1.0.0",
    "educoder/pest": "1.0.0",
    "league/oauth2-client": "dev-master"

In my Macosx environments with "port" I installed curl package:

    sudo port install php55-curl

These instructions are fine for every php environment with composer. In order to be more accurate I'm using Lumen microframework (Laravel).
